IOS10-HW13-IsakovAnton/Sourses/Cells/ArrowCell.swift

```diff
@@ -0,0 +1,102 @@
+
+import UIKit
+
+class ArrowCell: UITableViewCell {
+    
+    // MARK: - Outlets
+    let iconImageView: UIImageView = {
+        let imageView = UIImageView()
```

Comment:
```suggestion
    // MARK: - Outlets
    
    let iconImageView: UIImageView = {
        let imageView = UIImageView()
```
так же где прайваты у элементов? Почему я могу к ним обратиться извне?